pkg/virt-handler/vm.go

```diff
@@ -21,6 +22,18 @@ func NewVMController(listWatcher cache.ListerWatcher, domainManager libvirt.Doma
 				recorder.Event(vm, kubev1.EventTypeWarning, v1.SyncFailed.String(), err.Error())
 				return cache.ErrRequeue{Err: err}
 			}
+
+			restClient, err := kubecli.GetRESTClient()
+			if err != nil {
+				return cache.ErrRequeue{Err: err}
+			}
+			vm.Status.Phase = v1.Succeeded
+			err = restClient.Put().Resource("vms").Body(vm).
```

Comment:
There is currently only one way in kubernetes to go into the running state, and that is from `Pending`. We first have to check if the VM is currently in the `Pending` state. Even if that changes, how it is done now would mean that we would do a rest call to the api server on every VM update, even it is already in running state.